doc/related_projects.rst

```diff
@@ -182,21 +205,12 @@ and tasks.
   (`CRFsuite <http://www.chokkan.org/software/crfsuite/>`_ wrapper with
   sklearn-like API).
 
-- `skforecast <https://github.com/JoaquinAmatRodrigo/skforecast>`_ A python library
-  that eases using scikit-learn regressors as multi-step forecasters. It also works
-  with any regressor compatible with the scikit-learn API.
 
 **Deep neural networks etc.**
 
-- `nolearn <https://github.com/dnouri/nolearn>`_ A number of wrappers and
-  abstractions around existing neural network libraries
-
 - `Keras <https://www.tensorflow.org/api_docs/python/tf/keras>`_ High-level API for
   TensorFlow with a scikit-learn inspired API.
```

Comment:
this should be replaced with `scikeras`: https://github.com/adriangb/scikeras

Reply:
scikeras come at line 217.
From what I read in https://www.tensorflow.org/api_docs/python/tf/keras/Model, it seems fine to keep this entry. But no strong feeling.

Reply:
Keras has a scikit-learn inspired API, but it does not really work well with our meta-estimators. I am +1 on removing the Keras entry.